src/Resources/scaffolds/6.0/register/tests/Functional/User/RegisterTest.php

```diff
@@ -0,0 +1,159 @@
+            ->assertSee('Name is required')
+            ->use(Authentication::assertNotAuthenticated())
+        ;
+    }
+
+    /**
+     * @test
+     */
+    public function email_is_required(): void
+    {
+        $this->browser()
+            ->throwExceptions()
+            ->visit('/register')
+            ->assertSuccessful()
+            ->fillField('Name', 'Madison')
+            ->fillField('Password', 'password')
+            ->click('Register')
+            ->assertOn('/register')
+            ->assertSee('Email is required')
+            ->use(Authentication::assertNotAuthenticated())
+        ;
+    }
+
+    /**
+     * @test
+     */
+    public function email_must_be_email_address(): void
+    {
+        $this->browser()
+            ->throwExceptions()
+            ->visit('/register')
+            ->assertSuccessful()
+            ->fillField('Name', 'Madison')
+            ->fillField('Email', 'invalid-email')
+            ->fillField('Password', 'password')
+            ->click('Register')
+            ->assertOn('/register')
+            ->assertSee('This is not a valid email address')
+            ->use(Authentication::assertNotAuthenticated())
+        ;
+    }
+
+    /**
+     * @test
+     */
+    public function email_must_be_unique(): void
+    {
+        UserFactory::createOne(['email' => 'madison@example.com']);
+
+        $this->browser()
+            ->throwExceptions()
+            ->visit('/register')
+            ->assertSuccessful()
+            ->fillField('Name', 'Madison')
+            ->fillField('Email', 'madison@example.com')
+            ->fillField('Password', 'password')
+            ->click('Register')
+            ->assertOn('/register')
+            ->assertSee('There is already an account with this email')
+            ->use(Authentication::assertNotAuthenticated())
+        ;
+    }
+
+    /**
+     * @test
+     */
+    public function password_is_required(): void
+    {
+        $this->browser()
+            ->throwExceptions()
+            ->visit('/register')
+            ->assertSuccessful()
+            ->fillField('Name', 'Madison')
+            ->fillField('Email', 'madison@example.com')
+            ->click('Register')
+            ->assertOn('/register')
+            ->assertSee('Please enter a password')
+            ->use(Authentication::assertNotAuthenticated())
+        ;
+    }
+
+    /**
+     * @test
+     */
+    public function password_must_be_min_length(): void
+    {
+        $this->browser()
+            ->throwExceptions()
+            ->visit('/register')
+            ->assertSuccessful()
+            ->fillField('Name', 'Madison')
+            ->fillField('Email', 'madison@example.com')
+            ->fillField('Password', '1234')
+            ->click('Register')
+            ->assertOn('/register')
+            ->assertSee('Your password should be at least 6 characters')
+            ->use(Authentication::assertNotAuthenticated())
+        ;
+    }
```

Comment:
There's a lot of tests here too. What about a "success test" and a "validation error test" only? We would only be able to cover, for example, that "name" is required, "email" is an email and password requires 6 chars, but that's probably good enough.